libs/template/config.go

```diff
@@ -36,6 +39,22 @@ func newConfig(ctx context.Context, schemaPath string) (*config, error) {
 	}, nil
 }
 
+func validateSchema(schema *jsonschema.Schema) error {
+	for _, v := range schema.Properties {
+		switch v.Type {
```

Comment:
could v.Type be a `bool` instead of `boolean`?

Reply:
bool is not supported, adding a cast for that too.